src/Asset/EntrypointLookupInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Symfony\WebpackEncoreBundle\Asset;
```

Comment:
Missing license header. Rebase to the current master and run `php vendor/bin/php-cs-fixer fix` and it'll just handle all of this automatically